projects/Rockchip/options

```diff
@@ -0,0 +1,86 @@
+################################################################################
+# setup system defaults
+################################################################################
+
+  # Bootloader to use (syslinux / u-boot / bcm2835-bootloader)
+    BOOTLOADER="u-boot"
+
+  # Kernel extra targets to build
+    KERNEL_UBOOT_EXTRA_TARGET=""
+
+  # Kernel to use. values can be:
+  # default:  default mainline kernel
+    LINUX="${LINUX:-rockchip-4.4}"
+
+  # Fix linux rockchip-4.4 compile
+    if [ "$1" = "linux" -a "$LINUX" = "rockchip-4.4" ]; then
+      export KCFLAGS="-Wno-misleading-indentation -Wno-tautological-compare -Wno-unused-function -Wno-duplicate-decl-specifier -Wno-switch-unreachable"
+    elif [ "$1" = "RTL8192CU" -a "$LINUX" = "rockchip-4.4" ]; then
+      export KCFLAGS="-Wno-misleading-indentation -Wno-aggressive-loop-optimizations -Wno-int-to-pointer-cast -Wno-int-conversion"
+    elif [ "$1" = "RTL8192EU" -a "$LINUX" = "rockchip-4.4" ]; then
+      export KCFLAGS="-Wno-misleading-indentation"
+    elif [ "$1" = "RTL8812AU" -a "$LINUX" = "rockchip-4.4" ]; then
+      export KCFLAGS="-Wno-date-time -Wno-switch-unreachable"
+    elif [ "$1" = "media_build" -a "$LINUX" = "rockchip-4.4" ]; then
+      export KCFLAGS="-Wno-unused-variable -Wno-misleading-indentation -Wno-switch -Wno-declaration-after-statement -Wno-unused-function -Wno-unused-label -Wno-unused-result"
+    fi
```

Comment:
Hmm. What is the reason for adding this here, rather than in the relevant package - is it to reduce maintenance overhead by keeping this logic in one place (that's the only reason that comes to mind). I'm not entirely sure it's the right/best thing to do, but I guess it will work - it just "feels" wrong.

And `media_build` is no more.

Reply:
Thanks @MilhouseVH, I added these because it was the easiest to get kernel and modules to compile at the time when the project was new, will look into a proper solution and remove these overrides.